packages/auth/src/app.ts

```diff
@@ -290,7 +290,7 @@ export class App {
 
     // Grant Cancel
     router.delete<DefaultState, GrantRevokeContext>(
-      '/continue/:id',
+      '/:tenantId/continue/:id',
```

Comment:
What are we doing with this `tenantId`, or why do we need to include it here? I dont see it used in the `grantRoutes.revoke` method and I dont think it's used by the middleware. Should we be validating it in the `revoke` method? I do see we're checking it in POST /:tenantId/continue/:id (`grantRoutes.continue`).